tshirts.c

```diff
@@ -15,7 +15,11 @@ char size(int cms) {
 
 int main() {
     assert(size(37) == 'S');
+    assert(size(38) == 'S');

```

Comment:
You'll need to make up your mind if 38 is S or M... Otherwise one of the asserts will keep failing.
Use asserts to express your functionality.